backend/functions/lib/adapters/types/case.attorney.assignment.ts

```diff
@@ -0,0 +1,21 @@
+import { CaseAssignmentRole } from './case.assignment.role';
+
+export class CaseAttorneyAssignment {
+  _assignmentId: number;
+  _caseId: string;
+  _caseTitle: string;
+  _professionalId: string;
+  _role: CaseAssignmentRole;
```

Comment:
These are not private and you are accessing them in another test.

Reply:
Done, updated the naming on the class variables, based on our meeting discussion to use the google guidelines going forward.